### DISCUSSION TITLE
TE2 Command Line installation

### DISCUSSION THREAD

**Discussion by fausttiger007** [TabularEditor/TabularEditor3#628] [🙋♀️TE3 Q&A] [answered]
"The command line script above assumes you have Tabular Editor 2.x installed. The installation location of Tabular Editor 2.x should also be specified as part of your PATH environment variable."

As TE2 and TE3 are both  "tabulareditor.exe"   , how do you configure for any conflict if both TE2 and TE3 are co-installed and in path?

TE 3.3.4

Tony

**Comment by otykier** [comment, marked as answer]
You should not install both tools within the same folder. By default, TE2 is installed in c:\Program Files (x86)\Tabular Editor, while TE3 is in c:\Program Files\Tabular Editor 3.

If you’re talking about using the PATH variable, then you should only do that for TE2, as TE3 does not have a CLI yet. Once it does, the CLI executable will have a different name, something like te3cli.exe perhaps.
